views/beer.hbs

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>Im beer</div>
+
+    {{#each beersFromApi}} 
+    <div><img src="{{this.image_url}}"></div>
```

Comment:
watch indentation, these lines should be indented on space to the side inside the #each